src/app/core/component.ts

```diff
@@ -301,7 +301,7 @@ export default class Component {
             // gets teh feature defined by the id
             feature = ComponentUtils.getFeatureFromID(this._featureIDs[i]);
             console.log(feature);
-            renderedfeature = FeatureRenderer2D.renderFeature(feature);
+            renderedfeature = FeatureRenderer2D.renderFeature(feature, undefined);
```

Comment:
We shouldn't be passing undefined here. It'll cause a lot of errors later on. Pass `None` instead. Ideally the method definition should change rather than passing it manually.